src/domain/events/eventSearchForm/EventSearchForm.tsx

```diff
@@ -170,57 +177,69 @@ const EventSearchForm = ({
-                    options={languageOptions}
-                  />
-                </div>
+                <>
+                  <div className={styles.filtersRow4}>
+                    <Field
+                      hideLabel
+                      name="categories"
+                      className={styles.selectInput}
+                      component={MultiDropdownField}
+                      label={t('events:search.labelCategories')}
+                      placeholder={t('events:search.labelCategories')}
+                      clearButtonAriaLabel={t(
+                        'events:search.accessibility.categoryDropdown.clearButtonAriaLabel'
+                      )}
+                      selectedItemRemoveButtonAriaLabel={t(
+                        'events:search.accessibility.categoryDropdown.selectedItemRemoveButtonAriaLabel'
+                      )}
+                      options={categoryKeywords}
+                    />
+                    <Field
+                      hideLabel
+                      name="additionalCriteria"
+                      className={styles.selectInput}
+                      component={MultiDropdownField}
+                      label={t('events:search.labelActivities')}
+                      placeholder={t('events:search.labelActivities')}
+                      clearButtonAriaLabel={t(
+                        'events:search.accessibility.activityDropdown.clearButtonAriaLabel'
+                      )}
+                      selectedItemRemoveButtonAriaLabel={t(
+                        'events:search.accessibility.activityDropdown.selectedItemRemoveButtonAriaLabel'
+                      )}
+                      options={additionalCriteriaKeywords}
+                    />
+                    <Field
+                      component={DivisionSelectorField}
+                      className={styles.selectInput}
+                      name="divisions"
+                      showSearch={true}
+                      title={t('events:search.labelDivisions')}
+                      inputPlaceholder={t(
+                        'events:search.divisionInputPlaceholder'
+                      )}
+                    />
+                    <Field
+                      hideLabel
+                      name="inLanguage"
+                      className={styles.selectInput}
+                      component={MultiDropdownField}
+                      label={t('events:search.labelLanguage')}
+                      placeholder={t('events:search.labelLanguage')}
+                      options={languageOptions}
+                    />
+                  </div>
+                  <div className={styles.filtersRow4}>
+                    <div className={styles.checkboxWrapper}>
+                      <Field
+                        component={CheckboxField}
+                        name="isFree"
+                        label={t('events:search.labelIsFree')}
+                        style={checkboxSearchInputStyles}
```

Comment:
Other fields in this component seem to be styled with `className`. Should this field be styled in the same way?